src/components/BackButton/BackButton.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <q-btn
     flat
-    color="gray-dim"
```

Comment:
This has changed the arrow's color on other views, what about adding a prop "color" with default gray-dim?
![imagen](https://user-images.githubusercontent.com/66505715/189361021-cd4252f0-5ec3-47d7-a0e6-d916bc044919.png)